{"name": "test_auth", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable\n  (Session info: headless chrome=101.0.4951.64)\nStacktrace:\n#0 0x561c20b70533 <unknown>\n#1 0x561c208cf07f <unknown>\n#2 0x561c209063a7 <unknown>\n#3 0x561c208fa4a9 <unknown>\n#4 0x561c20922f42 <unknown>\n#5 0x561c208f9d63 <unknown>\n#6 0x561c2092304e <unknown>\n#7 0x561c20936224 <unknown>\n#8 0x561c20922e33 <unknown>\n#9 0x561c208f871a <unknown>\n#10 0x561c208f9875 <unknown>\n#11 0x561c20bb4e1d <unknown>\n#12 0x561c20bb8751 <unknown>\n#13 0x561c20b9f07e <unknown>\n#14 0x561c20bb9388 <unknown>\n#15 0x561c20b93fe0 <unknown>\n#16 0x561c20bd5748 <unknown>\n#17 0x561c20bd58c8 <unknown>\n#18 0x561c20bef70d <unknown>\n#19 0x7f5de2f5c3ba <unknown>\n#20 0x7f5de29f5393 <unknown>", "trace": "tests/test_auth.py:12: in test_auth\n    main_page.authorization()\n        browser    = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0df146b29086005d6ec9da5bae090e79\")>\n        main_page  = <pages.MainPage.MainPageHelper object at 0x7fade81dca60>\npages/MainPage.py:29: in authorization\n    auth_button.click()\n        auth_button = <selenium.webdriver.remote.webelement.WebElement (session=\"0df146b29086005d6ec9da5bae090e79\", element=\"ad7a0ab4-9c1f-459f-94d2-6f139bb1fba9\")>\n        dialog_window = <selenium.webdriver.remote.webelement.WebElement (session=\"0df146b29086005d6ec9da5bae090e79\", element=\"223db1a7-6805-46d4-822d-6d370744cc46\")>\n        excape     = <selenium.webdriver.remote.webelement.WebElement (session=\"0df146b29086005d6ec9da5bae090e79\", element=\"6291f3dd-f631-435c-ac9a-c7d01f07c4b5\")>\n        self       = <pages.MainPage.MainPageHelper object at 0x7fade81dca60>\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py:89: in click\n    self._execute(Command.CLICK_ELEMENT)\n        self       = <selenium.webdriver.remote.webelement.WebElement (session=\"0df146b29086005d6ec9da5bae090e79\", element=\"ad7a0ab4-9c1f-459f-94d2-6f139bb1fba9\")>\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py:773: in _execute\n    return self._parent.execute(command, params)\n        command    = 'clickElement'\n        params     = {'id': 'ad7a0ab4-9c1f-459f-94d2-6f139bb1fba9', 'sessionId': '0df146b29086005d6ec9da5bae090e79'}\n        self       = <selenium.webdriver.remote.webelement.WebElement (session=\"0df146b29086005d6ec9da5bae090e79\", element=\"ad7a0ab4-9c1f-459f-94d2-6f139bb1fba9\")>\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:430: in execute\n    self.error_handler.check_response(response)\n        driver_command = 'clickElement'\n        params     = {'id': 'ad7a0ab4-9c1f-459f-94d2-6f139bb1fba9'}\n        response   = {'status': 400, 'value': '{\"value\":{\"error\":\"element not interactable\",\"message\":\"element not interactable\\\\n  (Sessio...\\\\n#18 0x561c20bef70d \\\\u003Cunknown>\\\\n#19 0x7f5de2f5c3ba \\\\u003Cunknown>\\\\n#20 0x7f5de29f5393 \\\\u003Cunknown>\\\\n\"}}'}\n        self       = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0df146b29086005d6ec9da5bae090e79\")>\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py:247: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable\nE     (Session info: headless chrome=101.0.4951.64)\nE   Stacktrace:\nE   #0 0x561c20b70533 <unknown>\nE   #1 0x561c208cf07f <unknown>\nE   #2 0x561c209063a7 <unknown>\nE   #3 0x561c208fa4a9 <unknown>\nE   #4 0x561c20922f42 <unknown>\nE   #5 0x561c208f9d63 <unknown>\nE   #6 0x561c2092304e <unknown>\nE   #7 0x561c20936224 <unknown>\nE   #8 0x561c20922e33 <unknown>\nE   #9 0x561c208f871a <unknown>\nE   #10 0x561c208f9875 <unknown>\nE   #11 0x561c20bb4e1d <unknown>\nE   #12 0x561c20bb8751 <unknown>\nE   #13 0x561c20b9f07e <unknown>\nE   #14 0x561c20bb9388 <unknown>\nE   #15 0x561c20b93fe0 <unknown>\nE   #16 0x561c20bd5748 <unknown>\nE   #17 0x561c20bd58c8 <unknown>\nE   #18 0x561c20bef70d <unknown>\nE   #19 0x7f5de2f5c3ba <unknown>\nE   #20 0x7f5de29f5393 <unknown>\n        exception_class = <class 'selenium.common.exceptions.ElementNotInteractableException'>\n        json       = <module 'json' from '/usr/lib/python3.9/json/__init__.py'>\n        message    = 'element not interactable\\n  (Session info: headless chrome=101.0.4951.64)'\n        response   = {'status': 400, 'value': '{\"value\":{\"error\":\"element not interactable\",\"message\":\"element not interactable\\\\n  (Sessio...\\\\n#18 0x561c20bef70d \\\\u003Cunknown>\\\\n#19 0x7f5de2f5c3ba \\\\u003Cunknown>\\\\n#20 0x7f5de29f5393 \\\\u003Cunknown>\\\\n\"}}'}\n        screen     = None\n        self       = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fade8284c70>\n        st_value   = '#0 0x561c20b70533 <unknown>\\n#1 0x561c208cf07f <unknown>\\n#2 0x561c209063a7 <unknown>\\n#3 0x561c208fa4a9 <unknown>\\n#...7 0x561c20bd58c8 <unknown>\\n#18 0x561c20bef70d <unknown>\\n#19 0x7f5de2f5c3ba <unknown>\\n#20 0x7f5de29f5393 <unknown>\\n'\n        stacktrace = ['#0 0x561c20b70533 <unknown>', '#1 0x561c208cf07f <unknown>', '#2 0x561c209063a7 <unknown>', '#3 0x561c208fa4a9 <unknown>', '#4 0x561c20922f42 <unknown>', '#5 0x561c208f9d63 <unknown>', ...]\n        status     = 'element not interactable'\n        value      = {'error': 'element not interactable', 'message': 'element not interactable\\n  (Session info: headless chrome=101.0.495... 0x561c20bd58c8 <unknown>\\n#18 0x561c20bef70d <unknown>\\n#19 0x7f5de2f5c3ba <unknown>\\n#20 0x7f5de29f5393 <unknown>\\n'}\n        value_json = '{\"value\":{\"error\":\"element not interactable\",\"message\":\"element not interactable\\\\n  (Session info: headless chrome=1...>\\\\n#18 0x561c20bef70d \\\\u003Cunknown>\\\\n#19 0x7f5de2f5c3ba \\\\u003Cunknown>\\\\n#20 0x7f5de29f5393 \\\\u003Cunknown>\\\\n\"}}'"}, "attachments": [{"name": "stdout", "source": "e1e8d1c7-8fea-486e-9264-3f400d73050d-attachment.txt", "type": "text/plain"}], "start": 1653900401013, "stop": 1653900405711, "uuid": "0f41ac21-9189-4594-b4f5-ede9caf38ce2", "historyId": "e701bae96da603ae7345ed8f71bbc3db", "testCaseId": "5306df55395c93b83e69abe887b683bd", "fullName": "tests.test_auth#test_auth", "labels": [{"name": "feature", "value": "Авторизация на платформе"}, {"name": "severity", "value": "blocker"}, {"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "fedora"}, {"name": "thread", "value": "19602-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}